src/pages/sylvia/macros/attributes/msg.mdx

```diff
@@ -89,3 +89,105 @@ fn some_query(&self, ctx: QueryCtx) -> SomeResult {
     Ok(SomeResponse)
 }
 ```
+
+## Handling replies
+
+<Callout>
+  To use attributes present in this paragraph pre-Sylvia `2.0.0`, you need to enable the `replies`
+  feature using the [`sv::features`](features) attribute.
+</Callout>
+
+You can use additional parameters `handlers` and `reply_on` in the `#[sv::msg(reply)]` attribute.
+
+```rust {1} template="sylvia-cw-storage-contract"
+#[sv::msg(reply, handlers=[remote_instantiated], reply_on=Success)]
+fn store_remote_address(
+    &self,
+    ctx: ReplyCtx,
+    #[sv::data(instantiate, opt)] data: Option<MsgInstantiateContractResponse>,
+    payload: Option<MyPayloadType>
+) -> StdResult<Response> {
+    Ok(Response::new())
+}
+```
+
+<Callout>
+  This approach requires the use of `sylvia::ctx::ReplyCtx` instaed of `sylvia::types::ReplyCtx` due
```

Comment:
```suggestion
  This approach requires the use of `sylvia::ctx::ReplyCtx` instead of `sylvia::types::ReplyCtx` due
```

Reply:
That's the plan. Here it's supposed to teach about the technicalities of usage. The whole usage would be too big and in the tutorial for Sylvia all will be explained